lintconfig.json

```diff
@@ -24,6 +24,12 @@
     "enableGC": true,
     "exclude": [
         "vendor",
+        "mixer/adapter/svcctrl",
+        "^security",
```

Comment:
any chance to tweak settings and make some code change to get it to pass, or at least be a bit more granular than the whole directory ?

Reply:
I just want to make this rather than waiting -- that's the agreement among me, @guptasu, and @geeknoid, otherwise several other lint errors are coming in again and again, and we can't enable the linter anytime soon.

Those excludes are in three parts (`mixer/adapter/svcctrl`, `security/`, and `mixer/pkg/il`) and we know their owners ( @manlinl , @JimmyCYJ , and @mandarjog respectively). I think the communication cost regarding lint suppression / re-enabling will be low enough.

Reply:
note: those lint errors are already tracked as #1446, #1802, and #1832.